src/js/step.js

```diff
@@ -295,7 +295,7 @@ export class Step extends Evented {
    * @return {*|boolean} True if the step is open and visible
    */
   isOpen() {
-    return this.el && !this.el.hidden;
+    return !!(this.el && !this.el.hidden);
```

Comment:
I've kind of feel `Boolean()` reads better. Why do I feel like @rwwagner90 has an opinion on this one...

Reply:
I agree with @chuckcarpenter here, `Boolean` is much more readable than `!!`

Reply:
We should also change `@return {*|boolean}` to be `@return {boolean}` since we are ensuring it's always boolean now.